lms/templates/main.html

```diff
@@ -177,6 +177,15 @@
   </script>
 % endif
 
+<% gtm_id = static.get_value("GOOGLE_TAG_MANAGER_ID", getattr(settings, "GOOGLE_TAG_MANAGER_ID", None)) %>
```

Comment:
```suggestion
<% gtm_id = static.get_value("GOOGLE_TAG_MANAGER_ID", settings.GOOGLE_TAG_MANAGER_ID) %>
```

There is no reason to use `getattr` when we are certain that `settings.GOOGLE_TAG_MANAGER_ID` will at least be defined with a default value of None. In fact, it is harmful to use `getattr`, because it would silence errors in the case where `"GOOGLE_TAG_MANAGER_ID"` is typo'd, renamed, or removed.

This applies in general when writing Python -- always prefer accessing the attribute directly, only use `getattr` when absolutely necessary.

Reply:
Thank you, fixed